vortex-array/src/compute/compare.rs

```diff
@@ -22,6 +22,21 @@ pub enum Operator {
     Lte,
 }
 
+pub trait OperatorImpl<T> {
+    const FN_: fn(T, T) -> bool;
```

Comment:
Why isn't this `fn eval(lhs: &T, rhs: &T) -> bool` ?

Reply:
So I cann write 

this
```
impl<T: PartialOrd> OperatorImpl<T> for Lt {
    const FN_: fn(&T, &T) -> bool = PartialOrd::lt;
}
```
over
```
impl<T: PartialOrd> OperatorImpl2<T> for Lt {
    fn eval(t: &T, t2: &T) -> bool {
        PartialOrd::lt(t, t2)
    }
}
```